### PR TITLE
LPS-149318 Add error message when the field is empty

### PR DESCRIPTION
**Pull request by MariMalta**

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js [hunk 1]
```diff
@@ -297,12 +297,11 @@ const NotificationsInfo = ({
...
-		if (recipientType === 'roleType') {
-			const sectionsData = [];
-
-			const recipients =
-				selectedItem.data.notifications.recipients[notificationIndex];
+		let sectionsData = [];
 
+		const recipients =
+		selectedItem.data.notifications && selectedItem.data.notifications.recipients[notificationIndex];
+		if (recipients && recipientType === 'roleType') {
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js:305]: @MariMalta Please, add a blank line separating the definition and usage of recipients :)

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseUser.js [hunk 2]
```diff
@@ -66,6 +69,27 @@ const BaseUser = ({
 		setSearch('');
 	};
 
+	const checkSearchErrors = (errors, user) => {
+		const temp = errors.user ? [...errors.user] : [];
+		if (!temp[notificationIndex]) {
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseUser.js:74]: @MariMalta Please, add a blank line separating the definition and usage of temp

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
ci:stop

**Comment by MariMalta** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]

**Pull request closed without merging**